charts/tidb-backup/values.yaml

```diff
@@ -56,6 +56,9 @@ backupOptions: "--verbose=3"
 initialCommitTs: ""
 # restoreOptions is the options of loader https://www.pingcap.com/docs-cn/tools/loader/
 restoreOptions: "-t 16"
+# The time limit during which data is retained for each GC when backup, in the format of Go Duration.
+# When a GC happens, the current time minus this value is the safe point.
+tikvGCLifeTime: 3h
```

Comment:
I think it would better to use a long gc life time that impossible to reach as the default, e.g. 720h, which effectively "disable" the GC when backup in progress.